### DISCUSSION TITLE
devops/test/hub/11.0.3/docs/topics/c_install_ubuntu

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#91] [Q&A]
# devops/test/hub/11.0.3/docs/topics/c_install_ubuntu

You can find information about the tasks that you can perform to install HCL DevOps Test Hub (Test Hub) on Ubuntu.

https://help.hcl-software.com/devops/test/hub/11.0.3/docs/topics/c_install_ubuntu.html

**Comment by virtualmonster** [comment]
It would be nice to provide a readable form of the readme.md file here in the doc/browser
